cli/src/utils/addPackageDependency.ts

```diff
@@ -22,10 +22,8 @@ export const addPackageDependency = (opts: {
     const version = dependencyVersionMap[pkgName];
 
     if (devMode) {
-      //eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
eeehrm I ran `lint --fix` and it removed these, linter passes but editor complains 🤔 
<img width="1158" alt="CleanShot 2023-01-03 at 12 38 24@2x" src="https://user-images.githubusercontent.com/51714798/210350200-e15c72df-b614-48fb-86dc-f1dcf53cc915.png">

Reply:
Hmm... I don't know why this is happening, but my first guess something spooky with the circular dependency:

1. `addPackageDependency.ts` imports `dependencyVersionMap` from `installers/index.ts`
2. `installers/index.ts` imports `nextAuthInstaller` from `nextAuth.ts`
3. `nextAuth.ts` imports `addPackageDependency` from `addPackageDependency.ts`

Circular dependencies are a no-no in general and especially with TypeScript tooling. I moved the map to its own file (`713c5c3`) and... still get the `any`s. But at least they're on the command-line now too!

Reply:
Yeah I'm stuck here. TypeScript's type checker is still giving typescript-eslint an `any` type even after clearing caches / renaming a bunch of stuff around. 

On the surface this issue is behaving similarly to https://github.com/typescript-eslint/typescript-eslint/issues/4689 -> https://github.com/microsoft/TypeScript/issues/48313. So I'd suggest waiting until TypeScript 5 is released, and checking if the `eslint-disable`s added in 73ed575 are still needed. If they are, then this deserves its own bug - probably on typescript-eslint.